modules/access_tokens_guard/guard.ts

```diff
@@ -126,8 +126,8 @@ export class AccessTokensGuard<UserProvider extends AccessTokensUserProviderCont
    */
   #getBearerToken(): string {
     const bearerToken = this.#ctx.request.header('authorization', '')!
-    const [, token] = bearerToken.split('Bearer ')
-    if (!token) {
+    const [type, token] = bearerToken.split(' ')
```

Comment:
Apparently this should be one or more space characters as valid (there's been discussion about this on the IETF OAuth WG mailing list recently

Reply:
Can you share what the discussion has been?

Reply:
The discussion was based on the syntax in [RFC 6750](https://datatracker.ietf.org/doc/html/rfc6750#section-2.1):

```
     b64token    = 1*( ALPHA / DIGIT /
                       "-" / "." / "_" / "~" / "+" / "/" ) *"="
     credentials = "Bearer" 1*SP b64token
```

Where the `1*SP` refers to `1*element` for one or more elements, where `element` is `SP` which is `%x20` from [ABNF](https://datatracker.ietf.org/doc/html/rfc5234)

So technically, there can be more than one space between Bearer and the token value. In the OAuth 2.1 draft, the `Bearer` is explicitly relaxed to be case-insensitve: https://drafts.oauth.net/oauth-v2-1/draft-ietf-oauth-v2-1.html#section-5.1.1

The full discussion thread is here: https://mailarchive.ietf.org/arch/msg/oauth/8L3aqxz4tNICjrJXgUZvO3Oy5XY/